core-feature-pack/src/main/resources/configuration/standalone/subsystems.xml

```diff
@@ -5,5 +5,6 @@
       <subsystem>logging.xml</subsystem>
       <subsystem>jmx.xml</subsystem>
       <subsystem>request-controller.xml</subsystem>
+      <subsystem>security-manager.xml</subsystem>
```

Comment:
Having this here will make security manager subsystem enabled by default for all configuration build from this core.
Which would mean that we enable SM by default in all our shipped configurations.
I think this should be removed.